features/shared/checkout_previous_branch_after_success/new_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that creates a new current branch intact and leaves the previous branch intact
```

Comment:
Feature: Git Town commands that create a new branch set the current branch as the previous Git branch